packages/core/src/components/popover/popover.tsx

```diff
@@ -330,8 +319,6 @@ export class Popover<
             targetTagName = "div";
         }
 
-        const ref = mergeRefs(popperChildRef, this.targetRef);
```

Comment:
no longer need to merge refs, since react-popper is already doing that under the hood: https://github.com/floating-ui/react-popper/blob/beac280d61082852c4efc302be902911ce2d424c/src/Reference.js#L17-L23